### DISCUSSION TITLE
buenas no puedo abrir la aplicacion Firefly

### DISCUSSION THREAD

**Discussion by Jo6j9** [iotaledger/firefly#8623] [Help & Support]
conecto el ledger, intento abrir la aplicación  y no me deja sale este mensaje

, 17 Oct 2024 20:19:54 GMT
wallet: `local time 1729196394 doesn't match the time of the latest milestone timestamp: 1729196077`
por favor alguien me puede ayudar

**Comment by begonaalvarezd** [comment]
El error que estas experimento es a causa de tener la hora de tu ordenador mal, sincroniza el reloj de manera automática o con sincronización manual y ya deberías poder entrar
Si sigues teniendo problemas puedes volver a abrir el issue
